src/grader/grader_cli.ml

```diff
@@ -67,12 +72,22 @@ let remove_trailing_slash s =
   let len = String.length s in
   if len <> 0 && s.[len-1] = '/' then String.sub s 0 (len-1) else s
 
+let read_file f =
```

Comment:
Since the grader uses Lwt extensively, Lwt_io should be used to read files instead of Pervasives, which should look like : 
```
Lwt_io.with_file ~mode:Lwt_io.Input f Lwt_io.read >>= fun ic ->
Lwt_io.read ic >>= fun content ->
Lwt.return content
```
`with_file` ensures the file descriptor is closed after reading the file.